tensorflow/examples/learn/resnet.py

```diff
@@ -141,13 +135,14 @@ def res_net(x, y, activation=tf.nn.relu):
   return learn.models.logistic_regression(net, y)
 
 # Download and load MNIST data.
-mnist = input_data.read_data_sets('MNIST_data')
+mnist = learn.datasets.load_dataset('mnist')
 
-# Restore model if graph is saved into a folder.
-if os.path.exists('models/resnet/graph.pbtxt'):
-  classifier = learn.TensorFlowEstimator.restore('models/resnet/')
+# Create a new resnet classifier.
+classifier = learn.TensorFlowEstimator(
```

Comment:
We are trying to get rid of the (deprecated) TensorFlowEstimator. Can you rewrite this as an Estimator instead? It's a fairly straightforward translation -- include the optimize_loss call in your model_fn, which has to return also a train_op (which the optimize_loss call creates for you).